Source/BottomView/StackView.swift

```diff
@@ -7,22 +7,22 @@ protocol ImageStackViewDelegate: class {
 
 class ImageStackView: UIView {
 
-  struct Dimensions {
+  enum Dimensions {
```

Comment:
While empty enum cannot be created, enum itself represents a choice. I'd prefer a struct here